.github/workflows/cpp-packaging.yml

```diff
@@ -380,7 +405,7 @@ jobs:
         shell: bash
         run: |
           find firebase-cpp-sdk-*-package -type f
-      - name: upload artifacts
+      - name: upload SDK zip
```

Comment:
nit: Minor note on formatting. There is an empty line between steps in some places and not in others.
I personally feel its easier to read with an empty line in between but dont mind either format.

Reply:
Done, I fixed all the steps throughout the file to have blank lines.